### ISSUE TITLE
Problem configuring cinder-volume for two different storagebackends in different locations

### ISSUE THREAD

**Issue by DeadRabb1t** [osism/issues#261]
Hey there,

actually I am facing issues in configuring local storage and rbd storage in parallel inside of cinder.

The issue is, that cinder-volume configuration seems to not separate between lvm enabled and ceph enabled.
So I am getting a cinder-volume:rbd service on all compute nodes, which means that an outtage could make a volume unconfigurable, if volume was scheduled on that host.

f.e.:
```
+---------------+-----------------------------------------+--------+---------+-------+----------------------------+
| Binary        | Host                                    | Zone   | Status  | State | Updated At                 |
+---------------+-----------------------------------------+--------+---------+-------+----------------------------+
| cinder-volume | compute001.openstack.cluster@rbd-1 | cluster | enabled | up    | 2022-07-12T12:37:12.000000 |
| cinder-volume | compute001.openstack.cluster@lvm-1 | cluster | enabled | up   | 2022-07-12T12:37:08.000000 |
| cinder-volume | compute003.openstack.cluster@rbd-1 | cluster | enabled | up    | 2022-07-12T13:54:47.000000 |
| cinder-volume | compute003.openstack.cluster@lvm-1 | cluster | enabled | up    | 2022-07-12T13:54:48.000000 |
| cinder-volume | compute002.openstack.cluster@rbd-1 | cluster | enabled | up    | 2022-07-12T13:54:46.000000 |
| cinder-volume | compute002.openstack.cluster@lvm-1 | cluster | enabled | up    | 2022-07-12T13:54:56.000000 |
| cinder-volume | rbd:cluster@rbd-1                        | cluster | enabled | up    | 2022-07-12T13:54:50.000000 |
+---------------+-----------------------------------------+--------+---------+-------+----------------------------+
```

I am able to use overlays to activate rbd with different settings on my controlplane, but I need to remove it anyhow from compute-node cinder-volume configuration

If you need more information, please tell me what you need.

Greetings,

Christian

**Comment by berendt** [comment]
Can you please paste your cinder-volume.conf from one of the nodes.

**Comment by DeadRabb1t** [comment]
I attached one conf from compute and one from control
IPs, passwords and stuff had been removed.

[cindervolumecompute.txt](https://github.com/osism/issues/files/9109728/cindervolumecompute.txt)
[cindervolumecontrol.txt](https://github.com/osism/issues/files/9109729/cindervolumecontrol.txt)

**Comment by berendt** [comment]
This can probably also be closed after osism/issues#292.
